autogpt/log_cycle/log_cycle_mixin.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+import os
```

Comment:
Can this be built into the logger class from logs.py that’s already imported everywhere?

Reply:
oh yeah I forgot to do that

Reply:
done, now this mixin is exclusively using logger